packages/recommend-core/src/__tests__/getFrequentlyBoughtTogether.test.ts

```diff
@@ -0,0 +1,26 @@
+import { createRecommendationsClient } from '../../../../test/utils';
```

Comment:
`createRecommendationsClient` shouldn't be necessary now that we have `createRecommendClient`.

Reply:
Make sense, is that what you meant? https://github.com/algolia/recommend/pull/51/commits/5440f666ea6613268ed8bdc4a594558d29ce408c

Reply:
Yep!